packages/metadata-common/src/MetadataWidget.tsx

```diff
@@ -129,7 +156,13 @@ export class MetadataDisplay<
   // Render display of metadata list
```

Comment:
As mentioned in a previous comment by @ptitzler we should update `renderExpandableContent` to ignore the `tags` section of the json. This can be handled in a follow up PR if desired though